Felienne/hedy
# [BUG FIX] Store used commands correctly (#1658)

**Description**
In this PR we fix an issue where the used commands of users are not correctly stored in the Achievements db. This is due to comparing the wrong session variable to verify if there are new commands used. This comparison would always result in `False`, never storing new data. We also fix an issue where the "Gotta Use Them All" achievement could be obtained by only using the commands of `hedy.HEDY_MAX_LEVEL` (in this case lvl 18), however it should include the commands available in all levels together.

**Fix for**
This PR fixes #1658

**How to test**
Locally run several programs and notice the values being stored in the `Achievements` database in `dev_database.json`. For the fanatic testers: use alle possible commands from level 1 till level 18 and notice that you achieve the "Gotta Use Them All" achievement. 

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
